cordapp-example/workflows-kotlin/src/test/kotlin/com/example/test/contract/CashContractTests.kt

```diff
@@ -0,0 +1,163 @@
+package com.example.test.contract
+
+import com.example.contract.CashContract
+import com.example.state.CashState
+import net.corda.core.identity.CordaX500Name
+import net.corda.testing.core.TestIdentity
+import net.corda.testing.node.MockServices
+import net.corda.testing.node.ledger
+import org.junit.Test
+
+class CashContractTests {
+    private val ledgerServices = MockServices(listOf("com.example.contract", "com.example.flow"))
+    private val owner = TestIdentity(CordaX500Name("MegaCorp", "London", "GB"))
+    private val newOwner = TestIdentity(CordaX500Name("MiniCorp", "New York", "US"))
+    private val iouValue = 9L
+
+    @Test
+    fun `transaction must include Create command`() {
+        ledgerServices.ledger {
+            transaction {
+                output(CashContract.ID, CashState(iouValue, owner.party))
+                fails()
+                command(owner.publicKey, CashContract.Commands.Create())
+                verifies()
+            }
+        }
+    }
+
+    @Test
+    fun `Create - There should be no input`() {
+        ledgerServices.ledger {
+            transaction {
+                input(CashContract.ID, CashState(iouValue, owner.party))
+                output(CashContract.ID, CashState(iouValue, owner.party))
+                command(owner.publicKey, CashContract.Commands.Create())
+                `fails with`("There should be no input.")
+            }
+        }
+    }
+
+    @Test
+    fun `Create - Cash value must be positive`() {
+        ledgerServices.ledger {
+            transaction {
+                output(CashContract.ID, CashState(-1L, owner.party))
+                command(owner.publicKey, CashContract.Commands.Create())
+                `fails with`("Cash value must be positive")
```

Comment:
To absolutely confirm that it was the amount the problem, on top of the error message you can do:

```kotlin
command(owner.publicKey, CashContract.Commands.Create())
tweak {
    output(CashContract.ID, CashState(-1L, owner.party))
    `fails with`("Cash value must be positive")
}
output(CashContract.ID, CashState(1L, owner.party))
verifies()
```